selfapi.js

```diff
@@ -405,21 +423,30 @@ API.prototype = {
       html += '<h2 id="' + getAnchor(title) + '">' + title + '</h2>\n';
       html += '<pre>' + method.toUpperCase() + ' ' + fullPath + '</pre>\n';
       if (handler.description) {
-        html += '<p>' + handler.description + '</p>\n';
+        html += '<p>' + handler.description.replace(/\n/g, '<br>') + '</p>\n';
       }
 
       if (handler.examples && handler.examples.length > 0) {
         var example = handler.examples[0];
 
         var request = example.request || {};
-        if (request.headers || request.body) {
+        if (request.headers || request.body || request.urlParameters) {
           html += '<h3>Input</h3>\n<pre>';
+
+          // Format example-specific URL.
+          var exampleURL = method.toUpperCase() + ' ' + fullPath;
+          var requestParameters = request.urlParameters || {};
+          for (var parameter in requestParameters) {
+            exampleURL = exampleURL.replace(':' + parameter, requestParameters[parameter]);
+          }
+          html += exampleURL + '\n';
```

Comment:
Nit: Is it really useful to include an example URL when there are no URL parameters? Maybe it would be good to only do that `if (request.urlParameters)`?